core/src/main/scala/cats/std/tuple.scala

```diff
@@ -0,0 +1,15 @@
+package cats
+package std
+
+trait TupleInstances extends Tuple2Instances
```

Comment:
Since it looks like we're doing https://github.com/typelevel/cats/issues/800 I'll be closing that